CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### 6.10-SNAPSHOT
 
 #### Bugs
+* Fix #5560: bump com.marcnuri.plugins:gradle-api-maven-plugin to v0.0.5, allows compilation behind a proxy
+* Fix #5564: remove redundant lombok.Setter and lombok.experimental.Accessors annotations
+* Fix #5565: bump sundrio to 0.101.3 to fix Visitor problems
```

Comment:
is normal that it covers other fixes?

Reply:
No, I added the changelog for the rest of things that will go into 6.9.2.
I want to get started with the release ASAP